frontend/src/components/SearchBarFilter.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable array-callback-return */
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+import React, { useState } from 'react'
+import { Tag } from './Tag'
+import './searchbar.css'
+import { type Friend } from './PlanForm'
```

Comment:
this is interesting syntax to me since I normally see it as `import type {Friend} from '...'`

Reply:
yeah I have to put the `type` there or else eslint will yell at me like my parents do (childhood trauma).